react-single-page-app/README.md

```diff
@@ -0,0 +1,31 @@
+# React single-page app (SPA) | Sign in users, call protected API | Microsoft identity platform
+
+This minimal React application demonstrates usage of the Microsoft Authentication Library for React (MSAL React) to:
+
+- Sign in Azure AD users (authentication)
+- Call a protected web API (authorization)
+- Sign out users
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available scripts
+
```

Comment:
Can you add to the backlog that this needs to be updated to the new style?  Not a blocker for this import, but should be done soon.